internal/system/agent/init.go

```diff
@@ -43,6 +43,7 @@ var services = map[string]string{
 	internal.ExportDistroServiceKey: "Distro",
 	internal.SupportLoggingServiceKey: "Logging",
 	internal.SupportSchedulerServiceKey: "Scheduler",
+	internal.ConfigSeedServiceKey: "Config",
```

Comment:
- The `config-seed` service had intentionally been left out (as another service with which the SMA could also interact), the idea being that `config-seed` service is ephemeral, being launched at the outset to seed `consul` and shortly thereafter tearing itself down, thus thereafter remaining unavailable.
- While I'm not opposed to adding the `config-seed` service back, as I noticed you have per this PR, let's discuss as I want to hear your thoughts on the merits of doing so.

Reply:
I added it because the Docker files were looking for it and it didn't seem to do any harm in init. I'll remove it tomorrow.

Reply:
- The more thought that I gave to `config-seed` service having been intentionally been left out (as another service with which the SMA could also interact), the more I'm convinced that is should _remain_ left out (as another service with which the SMA could also interact). 
- Note that `config-seed` is not a _regular_ service and could be done by some other system/bootstrap capability in the future. 
- Yes, please remove it 👍